### DISCUSSION TITLE
Aapplication name proposal -  QLHighlight

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/SourceCodeSyntaxHighlight#63] [Ideas]
Github Repo name: SourceCodeSyntaxHighlight
App name: Syntax Highlight
Readme screenshot: Syntax Highlight Quicklook
Readme title: Quicklook extension for source files

---

## My suggestion - QLHighlight

It has a similar purpose with QLMarkdown. Both are Quick Look extensions. It would be better to change the application name to "QLHighlight" and update the repo name and documents accordingly.

Repo name: https://github.com/sbarex/QLHighlight
App name: QLHighlight

**Comment by sbarex** [comment]
The application is now over a year old ... I don't think it's appropriate to rename the executable. I will standardize the rest of the terminology.
